cvmfs/s3fanout.cc

```diff
@@ -773,7 +773,12 @@ string S3FanoutManager::GetContentType(const JobInfo &info) const {
     case JobInfo::kReqDelete:
       return "";
     case JobInfo::kReqPutCas:
-      return "application/octet-stream";
+    // Set correct content type for HTML files.
```

Comment:
I think I'd prefer adding another constant `kPutReqHtml` so that we don't need to look into the object details at this point.

Reply:
Yeah, that's much better. Thanks.